<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="launch_bbcf"/>
    <addaction name="separator"/>
    <addaction name="settings"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="view_palette"/>
    <addaction name="view_zoom"/>
   </widget>
   <widget class="QMenu" name="palettes_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Palettes</string>
    </property>
    <addaction name="import_palettes"/>
    <addaction name="export_palettes"/>
    <addaction name="separator"/>
    <addaction name="save_palette"/>
    <addaction name="save_palette_as"/>
    <addaction name="delete_palette"/>
    <addaction name="separator"/>
    <addaction name="copy_palette"/>
    <addaction name="paste_palette"/>
    <addaction name="separator"/>
    <addaction name="discard_palette"/>
   </widget>
   <widget class="QMenu" name="game_files_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Game Files</string>
    </property>
    <addaction name="apply_palettes"/>
    <addaction name="separator"/>
    <addaction name="restore_all"/>
    <addaction name="restore_character"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="tutorial"/>
    <addaction name="separator"/>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="cut_color"/>
    <addaction name="copy_color"/>
    <addaction name="paste_color"/>
    <addaction name="fill_color"/>
    <addaction name="swap_color"/>
    <addaction name="gradient_color"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="game_files_menu"/>
   <addaction name="palettes_menu"/>
   <addaction name="view_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="cut_color"/>
   <addaction name="copy_color"/>
   <addaction name="paste_color"/>
   <addaction name="fill_color"/>
   <addaction name="swap_color"/>
   <addaction name="gradient_color"/>
   <addaction name="separator"/>
   <addaction name="apply_palettes"/>
   <addaction name="restore_all"/>
   <addaction name="restore_character"/>
   <addaction name="separator"/>
   <addaction name="import_palettes"/>
   <addaction name="export_palettes"/>
   <addaction name="separator"/>
   <addaction name="save_palette"/>
   <addaction name="save_palette_as"/>
   <addaction name="delete_palette"/>
   <addaction name="separator"/>
   <addaction name="copy_palette"/>
   <addaction name="paste_palette"/>
   <addaction name="separator"/>
   <addaction name="discard_palette"/>
  </widget>
  <action name="exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/exit.ico</normaloff>:/images/images/exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="apply_palettes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/apply.ico</normaloff>:/images/images/apply.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Apply Palettes</string>
   </property>
   <property name="toolTip">
    <string>Apply all palettes to BBCF game data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="discard_palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/cancel.ico</normaloff>:/images/images/cancel.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Discard Changes</string>
   </property>
   <property name="toolTip">
    <string>Discard selected palette changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="restore_all">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/undo.ico</normaloff>:/images/images/undo.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Restore All</string>
   </property>
   <property name="toolTip">
    <string>Restore game data for all palettes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="restore_character">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/undo.ico</normaloff>:/images/images/undo.ico</iconset>
   </property>
   <property name="text">
    <string>Restore &amp;Character</string>
   </property>
   <property name="toolTip">
    <string>Restore game data for selected character palettes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="launch_bbcf">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/launch.ico</normaloff>:/images/images/launch.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Launch BBCF</string>
   </property>
   <property name="toolTip">
    <string>Launch BBCF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="view_palette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Palette Dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="view_zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom Dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="import_palettes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/import.ico</normaloff>:/images/images/import.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="export_palettes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/export.ico</normaloff>:/images/images/export.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="toolTip">
    <string>Export saved palettes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/settings.ico</normaloff>:/images/images/settings.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="toolTip">
    <string>Edit settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="copy_palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/copy.ico</normaloff>:/images/images/copy.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy current palette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="paste_palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/paste.ico</normaloff>:/images/images/paste.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste copied palette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="save_palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.ico</normaloff>:/images/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save current palette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="delete_palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/delete.ico</normaloff>:/images/images/delete.ico</iconset>
   </property>
   <property name="text">
    <string>D&amp;elete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected saved palette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="save_palette_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.ico</normaloff>:/images/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="toolTip">
    <string>Save current palette as</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="tutorial">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/info.ico</normaloff>:/images/images/info.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Tutorial</string>
   </property>
   <property name="toolTip">
    <string>Show tutorial dialog</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/question.ico</normaloff>:/images/images/question.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="cut_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/cut.ico</normaloff>:/images/images/cut.ico</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="toolTip">
    <string>Cut current selected color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="copy_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/copy.ico</normaloff>:/images/images/copy.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy current selected color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="paste_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/paste.ico</normaloff>:/images/images/paste.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="fill_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/fill.ico</normaloff>:/images/images/fill.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Fill</string>
   </property>
   <property name="toolTip">
    <string>Fill color selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="swap_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/swap.ico</normaloff>:/images/images/swap.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Swap</string>
   </property>
   <property name="toolTip">
    <string>Swap colors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="gradient_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/gradient.ico</normaloff>:/images/images/gradient.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Gradient</string>
   </property>
   <property name="toolTip">
    <string>Generate a color gradient</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
